### ISSUE TITLE
WDM caches version of installed browser for too long

### ISSUE THREAD

**Issue by asolntsev** [bonigarcia/webdrivermanager#483]
**Description of the problem**: 
WDM caches version of installed browser for 24 hours by default.
**It's too long!**

Every time when Chrome upgrades (e.g. from 81 to 83 today), people continue to get famous "this chromedriver supports only chrome 81" error **for a whole day**!

Look at file `~/.m2/repository/webdriver/resolution.properties`:

```
chrome=81
chrome-ttl=18\:21\:16 23/05/2020 EEST
```

(but my Chrome browser already upgraded to version 83). 

**Solution**:
I suggest to introduce another setting like `wdm.ttl.for-installed-browser=300` in addition to  `wdm.ttl=86400`. It's cheap to check the version of installed browser. It's ok to check it once in every 5 minutes or so. 

**Browser and version**: any
**Operating system**: any
**WebDriverManager version**: 4.0.0, 3.8.x

**WebDriverManager use**: 
```java
WebDriverManager.chrome().setup();
```

**Comment by ckempster** [comment]
Have also seen this issue,WDM caches version of installed browser for 24 hours by default.

**Comment by bonigarcia** [comment]
Thanks for reporting @asolntsev. Indeed maybe TTL=1day makes sense for driver versions, but perhaps it is too much for browser versions. Let me think about it (together with #484)

**Comment by bonigarcia** [comment]
This is available in WebDriverManager 4.1.0, just released.
